client.go

```diff
@@ -0,0 +1,32 @@
+package dp_redis
+
+import (
+	"github.com/go-redis/redis"
+	"time"
+)
+
+// Redis - structure for the redis client
+type Redis struct {
+	client *redis.Client
+	ttl    time.Duration
+}
+
+// Options - config options for the redis client
+type Options struct {
+	Addr     string
+	Password string
+	DB       int
+	ttl      time.Duration
```

Comment:
Should `ttl` be public? If not you won't be able to access it outside of this package.

Reply:
You're right, miss type on my part.